src/test/components/Agent/AgentInfo.test.tsx

```diff
@@ -0,0 +1,43 @@
+import { render } from '@testing-library/react'
+import AgentInfo from 'src/components/AgentInfo'
+import Agent from 'src/types/Agent'
+import Office from 'src/types/Office'
+
+const anytownOffice: Office = {
+  id: 0,
+  street: '123 Noendinsight Circle',
+  city: 'Anytown',
+  state: 'CA',
+  zip: '12345',
+  type: 'random',
+}
+const agentWithFirstName: Agent = {
+  firstName: 'Jebediah E.',
+  lastName: 'Hoover',
+  fullName: 'Jebediah E. Hoover',
+  office: anytownOffice,
+  id: '54321',
+  averageRating: 1,
+  type: 'human',
```

Comment:
I never noticed 'type' before in `Agent` or `Office` types. What is it for?

Reply:
Its supposed to designate what the data type is, e.g. 'Agent' or 'Office'. Useful for places like search results where it returns different types in the same list.

Reply:
Here's an example of how it's used, we check this value before type-casting it when building things like SearchResult:
```javascript
// SearchResult.tsx
export default function SearchResult({
  result,
  onClick,
  ...cardProps
}: ISearchResult) {
  return (
    <ListItem onClick={onClick} body {...cardProps}>
      {result.type === 'Agent' ? (
        <SearchResultAgent agent={result as Agent} />
      ) : (
        <SearchResultOffice office={result as Office} />
      )}
    </ListItem>
  )
}

```